pages/common/nc.md

```diff
@@ -4,7 +4,7 @@
 
 - Listen on a specified port:
 
-`nc  -l {{port}}`
+`nc  -l -p {{port}}`
```

Comment:
It seems to be a platform specific difference. Suggest that you split the page into 2 versions. One in linux and one in osx.